ui/lib/config-ui/addon/routes/messages/create.js

```diff
@@ -15,19 +16,31 @@ export default class MessagesCreateRoute extends Route {
     },
   };
 
+  async getMessages(authenticated) {
+    try {
+      return await this.store.query('config-ui/message', {
+        authenticated,
+      });
+    } catch {
+      return [];
+    }
+  }
+
   model(params) {
     const { authenticated } = params;
-
-    return this.store.createRecord('config-ui/message', {
-      authenticated,
+    return hash({
+      message: this.store.createRecord('config-ui/message', {
+        authenticated,
+      }),
+      messages: this.getMessages(authenticated),
     });
   }
 
   setupController(controller, resolvedModel) {
     super.setupController(controller, resolvedModel);
 
     controller.breadcrumbs = [
-      { label: 'Messages', route: 'messages', query: { authenticated: !!resolvedModel.authenticated } },
+      { label: 'Messages', route: 'messages', query: { authenticated: controller.authenticated } },
```

Comment:
curious why you changed this to the controller value? I think resolvedModel is more reliable (controllers have some weird behavior)

Reply:
Oh, it's because `authenticated` isn't being returned on the model from `model`. If you add that to the hash it'll be available on `resolvedModel` :)

Reply:
thank you!